VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationSelectView.java

```diff
@@ -96,21 +96,18 @@ public boolean isValidInput() {
 
         // check if this is required field and has an input value
         if (mField.isRequired() && !hasValue()) {
-            handleError(mField.getErrorMessage().getRequired());
+            String errorMessage = mField.getErrorMessage().getRequired();
+            if(errorMessage==null || errorMessage.isEmpty()){
+                errorMessage = getView().getResources().getString(R.string.error_select_field,
+                        mField.getLabel());
+            }
+            if(errorMessage!=null){
+                handleError(errorMessage);
+            }
```

Comment:
Null check not required, <code>handleError(String)</code> checks for null and empty text.